### PR TITLE
Connect WA-PHD to WA UAT (test)

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
Connect both staging and prod to WA UAT

This PR contains the settings for connecting staging and prod to WA UAT

## Changes
- Prod YML
- Staging YML

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #414 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

**Comment by carlosfelix2** [comment]
Should those new receivers be added to the organizations-prod.yml and organizations-test.yml files?

File: prime-router/settings/prod/0037-wa-uat.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - name: elr-neg
+    organizationName: wa-phd
+    topic: covid-19
+    jurisdictionalFilter:
+      - doesNotMatch(abnormal_flag, A)
+      - orEquals(ordering_facility_state, WA, patient_state, WA)
+    translation:
+      type: HL7
+      useBatchHeaders: true
+      nameFormat: washington_negative
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0037-wa-uat.yml:55]: I presume this naming difference is how the 'elr' and 'elr-neg' will be distinguised.

File: prime-router/settings/prod/0037-wa-uat.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    timing:
+      operation: "MERGE"
+      numberPerDay: 12
+      initialTime: "01:15"
+      timeZone: "EASTERN"
+      maxReportCount: 500
+    description: ""
+    transport:
+      type: AS2
+      receiverUrl: "https://uat-onehealthport-as2.axwaycloud.com/exchange/ZZOHPUAT"
```
Comment (jimduff-usds) [inline prime-router/settings/prod/0037-wa-uat.yml:42]: Just sanity checking:  this URL is the same for both regular elr and elr-neg.   Is that intended?

**Review by jimduff-usds — APPROVED** [review]
A couple minor sanity check questions for you.

It occurs to me that our "check" tool, that works quite well in checking for last mile issues with SFTPs, probably doesn't work for AS2, unless you did that.   It would be nice to have such a tool.

Also a minor note that we have a lot of stuff that wakes up at xx:15 - I've been asking Mo to spread this out a bit, but still try to run it on odd hours only.

**Review by carlosfelix2 — APPROVED** [review]
Tested by loading configuration locally

**Pull request merged**